### PR TITLE
Fix apigateway issues with terraform v0.12.21

### PR DESCRIPTION
**Pull request by duongpv7**
Some fixes:
* moto.apigateway.responses.APIGatewayResponse.resource_methods
 -> Add support for "DELETE" method
  (This is an issue when deleting an apigateway method)
* moto.apigateway.responses.APIGatewayResponse.integrations
-> Get "httpMethod" value from method resource, this "httpMethod" was not included in body request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,476 +1,3 @@
...
-```
-
-## Releases
-
-Releases are done from travisci. Fairly closely following this:
-https://docs.travis-ci.com/user/deployment/pypi/
-
-- Commits to `master` branch do a dev deploy to pypi.
-- Commits to a tag do a real deploy to pypi.
+Fork of https://github.com/spulec/moto/
```
Comment (bblommers) [inline README.md]: We'd like to keep the original README :)

File: moto/sqs/models.py [hunk 2]
```diff
@@ -98,10 +98,10 @@ def utf8(str):
             encoded += struct.pack(struct_format, len(data_type)) + utf8(data_type)
             encoded += TRANSPORT_TYPE_ENCODINGS[data_type]
 
-            if data_type == "String" or data_type == "Number":
+            if data_type == "String" or data_type.startswith("Number"):
                 value = attr["string_value"]
             elif data_type == "Binary":
-                print(data_type, attr["binary_value"], type(attr["binary_value"]))
+                # print(data_type, attr["binary_value"], type(attr["binary_value"]))
```
Comment (bblommers) [inline moto/sqs/models.py]: Let's remove this completely

File: setup.py [hunk 3]
```diff
@@ -67,8 +67,8 @@ def get_version():
 
 
 setup(
-    name='moto',
-    version=get_version(),
+    name='moto-ext',
```
Comment (bblommers) [inline setup.py]: Keep this file as is please

**Comment by bblommers** [comment]
Hi @whummer, just tagging you to check whether you're aware of this PR.

Looks like there are quite a few useful changes in here - a clean version is very welcome!

**Comment by whummer** [comment]
Thanks for letting me know @bblommers . Not all changes in this branch are ready and/or suitable to be merged into `moto` at this point. I'll work with @duongpv7 to cherry-pick only the changes that are supposed to be in this PR.

**Pull request closed without merging**